todo/src/components/ToDoList.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { connect } from "react-redux";
+import { addItem, toggleComplete } from "../actions";
+
+class ToDoList extends React.Component {
+  state = {
+    newItem: ""
+  };
+
+  handleChanges = e => {
+    e.persist();
+    this.setState({ newItem: e.target.value });
+  };
+
+  addItem = e => {
+    e.preventDefault();
+    this.props.addItem(this.state.newItem);
+    this.setState({ newItem: "" });
+  };
+
+  toggleComplete = (e, index) => {
+    e.preventDefault();
+    this.props.toggleComplete(index);
+  };
+
+  render() {
+    // console.log(this.props.items);
+    return (
+      <div className="to-do-container">
+        <React.Fragment>
```

Comment:
```suggestion

```
Don't need this Fragment here, you only need to use a Fragment when you don't want to return a parent div or other tag